src/components/apiDelegation/reusables/OverviewPageContent/OverviewPageContent.tsx

```diff
@@ -152,9 +152,14 @@ export const OverviewPageContent = ({
             </Button>
           </div>
         )}
-        <Panel title={t('api_delegation.card_title')}>
-          {t('api_delegation.api_panel_content')}
-        </Panel>
+        <div className={classes.infoPanelWrapper}>
+          <Panel
+            title={t('api_delegation.card_title')}
```

Comment:
Completely by accident, I discovered this nifty little thing called soft hyphen (U+00ad) the other day. It is an invisible character that only activates when the lack of space requires it to split up words. Maybe it can help you out here with the bane of existence that is 'Programmeringsgrensesnitt'? 🙂

Reply:
Superbra tips. Takk. Men regelen (i min bok i alle fall): dersom man trenger eksotisk css så er strukturen feil... ;)

Reply:
Det er jeg helt enig i 😅
Men dette er faktisk ikke css - det er en karakter man putter inn i teksten i tekstfila. Så om man har uhorvelige ord så kan man legge inn denne karakteren der det er naturlig å dele opp sånn at siden vet hvor man skal sette en bindestrek om det er nødvendig.
Godt mulig det finnes bedre løsninger der ute, men tenkte det var kjekt å dele tipset, i tilfelle det kunne komme til bruk 🙂